packages/assets/styles/classes.scss

```diff
@@ -1183,3 +1183,97 @@ select {
   border-top-left-radius: var(--radius-md) !important;
   border-top-right-radius: var(--radius-md) !important;
 }
+
+// Callouts
+
+.markdown-alert {
+  padding: 0.5rem 1rem;
+  margin-bottom: 16px;
+  color: inherit;
+
+  &::after {
+    position: absolute;
+    display: block;
+    content: '';
+    background-color: red;
+    width: 0.25em;
+    height: 15%;
```

Comment:
what is with these weird percentages and transforms? shouldn't the alert have position:relative and the pseudoelement be a top: 0, left: 0, bottom: 0? is there a particular reason it's done in this weird magic number-y way?